src/main/java/net/neoforged/neoforge/event/AddReloadListenerEvent.java

```diff
@@ -43,8 +52,25 @@ public void addListener(PreparableReloadListener listener) {
         listeners.add(new WrappedStateAwareListener(listener));
     }
 
+    /**
+     * Adds a listener just before the corresponding vanilla listener.
+     *
+     * @param vanillaListener represents the vanilla listener to add the listener before.
+     * @param listener the listener to add to the ResourceManager on reload
```

Comment:
Per the format check task:
```suggestion
     * @param vanillaListener represents the vanilla listener to add the listener before.
     * @param listener        the listener to add to the ResourceManager on reload
```
(Please run `spotlessApply` instead of manually fixing the error.)